### ISSUE TITLE
Newsfeed update stuck

### ISSUE THREAD

**Issue by amboris** [inorichi/tachiyomi#1420]
**App version:**
0.6.1-0.7.4
**Issue/Request:**
the new chapter update check bar on my newsfeed will allways stuck at the last manga (sources I use mangasee/kissmanga)
although I allways get another feed that tells me which manga has new chapter when the check reach the last one.

**Steps to reproduce (if applicable)**

 1. go to my libary
 2. swipe down to check for new chapters
 3. check stuck at last manga

**Other details:**

**Comment by inorichi** [comment]
Device and OS? Is this the same as #1487?

**Comment by amboris** [comment]
the description sounds to adress the same issue. i use android 7

**Comment by yoohoox8** [comment]
@amboris i have the similar issue i should also mention that on the Amazon tablet i can't cancel an update midway either. I have it on my S8 and it's working just fine. I only have issues with Amazon tablet

**Comment by rgarrigue** [comment]
Similar to #1428 ?

**Comment by amboris** [comment]
Similar to the commend "jkl1234567
commented 4 months ago

After a bit of research I have found that occasionally when the library is updating the update would get stuck on a chapter and would stay that way. This prevents the automatic update from triggering as well as the manual update as the library is still 'updating'. As I had notifications turned off previously I had no way to cancel the update and the library was unable to update. Currently I have left notifications on to monitor this issue. I have browsed the forums and there was a similar issue back in 2016 for an older version but I have noticed this issue amongst the versions that I have been using so the issue or a similar form of the issue still persists in these versions." From #1428
